### DISCUSSION TITLE
Marker icon SVG

### DISCUSSION THREAD

**Discussion by danieldanielecki** [react-native-maps/react-native-maps#4298] [Feature requests]
# Feature request

Allow an `icon` prop of the `<Marker />` component to accept `.svg` files.

## Why it is needed

We could have a scalable icon, which wouldn't lose quality on zoom in.

## Possible implementation

`<Marker icon={require("../assets/images/sample.svg")} />`

### Code sample

Currently, pointing to `.svg` file would display the default marker.
`<Marker icon={require("../assets/images/sample.svg")} />`

![screenshot](https://user-images.githubusercontent.com/12736263/164736365-a898d2d0-c425-425e-a2c0-805af00aabe5.png)

**Comment by galechus** [comment]
Hi @danieldanielecki,
have you tried:
```
<Marker>
    <View style={{ width: 40, height: 56 }}>
        <Svg width="100%" height="100%" viewBox="0 0 40 56">
            <Path d="M19.7 0c-10.9 .2-19.7 9.1-19.7 20v.1c0 .1 0 .2 0 .3c.1 7.6 4.5 14.1 10.7 17.4c1.8 .9 3.1 2.4 3.8 4.3l5.5 13.9l5.5-14c.7-1.8 2.1-3.3 3.8-4.2c6.4-3.4 10.7-10.1 10.7-17.8c0-11-9-20-20-20c-0.1 0-0.2 0-0.3 0Z" fill={fill} />
        </Svg>
    </View>
</Marker
```

**Reply by ahmedam55** [reply]
Thanks a lot!

**Comment by affansk** [comment]
@danieldanielecki  just convert SVG to react component and use it
